src/main/java/com/marginallyclever/makelangelo/Makelangelo.java

```diff
@@ -1091,4 +1099,20 @@ public static void main(String[] args) {
 			makelangeloProgram.run();
 		});
 	}
-}
\ No newline at end of file
+	
+	public String getFullVersionString(){
+		return getLiteVersionString()+ " ("+DETAILED_VERSION+")";		
+	}
+	public String getLiteVersionString(){
+		return Translator.get("TitlePrefix") + " " +VERSION;		
+	}
+	
+	private static Makelangelo currentInstance = null;
+	
+	public static String staticgetFullVersionString(){
```

Comment:
the word `static` should not appear in the name of the method, we already know it is `static`.
If you have trouble with a method called in a static way and with instance, maybe you should put the code you want in a new object. Here, it could be a `Version` object wtih at least  2 static methods: `getFullVersionString()` and `getLiteVersionString()`.
You should also put this new feature in a another branch as it is not related to the naming of turtle.